	uint32 id                       # id команды
	SMpu mpu               # Данные с датчика MPU
	SSensor laserL
	SSensor laserR
	SSensor uzi